File: iosApp/iosApp/DesignSystem/Components/EyebrowLabel.swift
import SwiftUI

/// Uppercase mono label used for screen eyebrows ("HOY · 28 ABRIL"),
/// section eyebrows ("ESTA SEMANA"), category labels, and metric captions.
///
/// Renders `text` verbatim — the caller decides casing. The 1.76 tracking
/// is applied here because `Font.custom(...)` cannot carry letter-spacing
/// on iOS; the Android counterpart bakes it into `labelSmall`.
struct EyebrowLabel: View {
    let text: String
    var color: Color = .fridgeyInkMuted

    var body: some View {
        Text(text)
            .font(FridgeyFont.labelSmall)
            .tracking(1.76)
            .foregroundStyle(color)
    }
}

File: composeApp/src/androidMain/kotlin/ule/jescuj00/fridgey/ui/components/EyebrowLabel.kt
package ule.jescuj00.fridgey.ui.components

import androidx.compose.material3.MaterialTheme
import androidx.compose.material3.Text
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier
import androidx.compose.ui.graphics.Color
import ule.jescuj00.fridgey.ui.theme.InkMuted

/**
 * Uppercase mono label used for screen eyebrows ("HOY · 28 ABRIL"),
 * section eyebrows ("ESTA SEMANA"), category labels, and metric captions.
 *
 * Renders `text` verbatim — the caller decides on casing; the tracking and
 * mono family come from `MaterialTheme.typography.labelSmall` (defined in
 * `Type.kt`).
 */
@Composable
fun EyebrowLabel(
    text: String,
    modifier: Modifier = Modifier,
    color: Color = InkMuted,
) {
    Text(
        text = text,
        style = MaterialTheme.typography.labelSmall,
        color = color,
        modifier = modifier,
    )
}
